Commencement.Mvc/Views/SpecialNeeds/Create.aspx

```diff
@@ -17,7 +17,7 @@
     <h2>Create</h2>
 
     <% using (Html.BeginForm()) {%>
-        <%: Html.ValidationSummary() %>
+        <%: Html.ValidationSummary(true,"Please correct all errors below") %>

```

Comment:
@jSylvestre do you really want the "true" here and to have it ignore property errors